### ISSUE TITLE
Right-click pasting of images not possible in visual mode

### ISSUE THREAD

**Issue by ronblum** [rstudio/rstudio#11128]
### System details

    RStudio Edition : Desktop
    RStudio Version : 1.4.1717, 2022.06.0-daily+341
    OS Version      : MacOS 12.4, Ubuntu 20.04.3, Windows 11
    R Version       : 3.6.3, 4.1.2

### Steps to reproduce the problem

1. Open an image, such as https://tinyurl.com/littlekittykat
2. Copy the image, e.g. with <kbd>Right-click -> Copy Image</kbd>.
3. In RStudio Desktop, open an R Markdown or Quarto document.
4. Switch to visual mode.
5. <kbd>Right-click</kbd> in the body of the document.
    - The "Paste" option is grayed out and unclickable.
6. <kbd>Cmd-V</kbd> (MacOS) or <kbd>Ctrl-V</kbd> (Linux & Windows)
    - The image is pasted.

### Describe the problem in detail

<kbd>Right-click -> Paste</kbd> is not available.

### Describe the behavior you expected

Pasting with a right-click should be possible.

Note: This has been around for several releases and is fixed by Electron, so this is only for reference.

- [ ] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [ ] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [ ] If I am reporting an RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [ ] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by matiasandina** [comment]
@ronblum Sorry to comment on a closed one, but I have been having this issue for a while. Multiple updates have not solved this (performed a couple of updates throughout 2022 and 2023). I am currently running this version:

```
─ Session info ─────────────────────────────────────────────────────────────────────────────────────────────────────────
 setting  value
 version  R version 4.1.2 (2021-11-01)
 os       Ubuntu 22.04.3 LTS
 system   x86_64, linux-gnu
 ui       RStudio
 language (EN)
 collate  en_US.UTF-8
 ctype    en_US.UTF-8
 tz       America/New_York
 date     2024-01-04
 rstudio  2023.09.1+494 Desert Sunflower (desktop)
 pandoc   2.9.2.1 @ /usr/bin/pandoc
 ```

The error is not exactly that the `Paste` option is unavailable. But when I actually click `Paste`, nothing happens.


https://github.com/rstudio/rstudio/assets/7494967/e597d41e-879a-4bb0-9cd2-3e1f5dada6e5

**Comment by ronblum** [comment]
@matiasandina No problem, thanks for pointing this out! I can reproduce the issue in 2023.09.1+494 (Desert Sunflower) on MacOS 14.3, but it works in 2023.12.0+369 (Ocean Storm). Possibly fixed due to #12690?

Would you be willing to try it out with the latest version of RStudio Desktop, 2023.12.0+369? I only checked this on MacOS, though, so if you prefer to wait till I try this on Ubuntu 22, that's cool.

**Comment by matiasandina** [comment]
Confirmed. Updated fixed the issue. Thanks!
